app/app.ts

```diff
@@ -151,6 +150,12 @@ function setupIpc(localSettings: LocalSettings, scrSettings: ScrSettings) {
     })
 
   localSettings.on('change', settings => {
+    const loginSettings = {
+      openAtLogin: settings.openAppAtLogin,
+    }
+    if (loginSettings !== app.getLoginItemSettings()) {
```

Comment:
In JavaScript, objects are compared by their reference so this condition will always be true. This should probably be rewritten to something like:
```ts
if (settings.openAppAtLogin !== app.getLoginItemSettings().openAtLogin) {
  //
}
```

Reply:
ah that's embarassing, i thought != was reference and !== was value 😅